app/src/styles.css

```
@@ -460,18 +460,39 @@ h1 {
   inset: 0;
 }
 
+.code-viewer__compile-output {
+  flex: none;
+  margin: 0;
+  padding: 0.5em 1em;
+  max-height: 8em;
+  overflow: auto;
+  border-top: 1px solid var(--color-border);
+  border-left: 3px solid transparent;
+  font-family: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)